src/test/java/com/exasol/reflect/ReflectionException.java

```diff
@@ -0,0 +1,13 @@
+package com.exasol.reflect;
```

Comment:
This is obvious duplication between projects. We need to reduce this soon.

Reply:
Yes. For now this is only about the migration. Once this is merged and done, we can work on it by steps.